src/main/java/by/issoft/mandrik/driver/DriverSingleton.java

```diff
@@ -0,0 +1,25 @@
+package by.issoft.mandrik.driver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.util.concurrent.TimeUnit;
+
+public class DriverSingleton {
```

Comment:
Press ctrl+alt+L for this class :)

Reply:
Rename this class, it is not singleton (rename into DriverHolder f.e.). Or you can ask me, and I can say, what is incorrect in your singleton